diff --git a/node_modules/reicon-react/createIcon.js b/node_modules/reicon-react/createIcon.js
index 937c9b6..e424110 100644
--- a/node_modules/reicon-react/createIcon.js
+++ b/node_modules/reicon-react/createIcon.js
@@ -42,7 +42,7 @@ const createIcon = (displayName, iconData) => {
         viewBox: '0 0 24 24',
         fill: 'none',
         className: className ? 'reicon ' + className : 'reicon',
-        style: { color, ...style },
+        style: { ...(color !== 'currentColor' ? { color } : {}), ...style },
         ...rest,
         dangerouslySetInnerHTML: { __html: html },
       });
